templates/event/show.html.twig

```diff
@@ -4,56 +4,38 @@
 
 {% block content %}
     <div class="row">
-        <div class="col-12">
+        <div class="col-6">
             <h1>Event</h1>
         </div>
+        <div class="col-6 text-right">
+            <a class="btn btn-warning" href="{{ path('event_unselect') }}">
+                Leave and go back to events
+                <i class="fa fa-sign-out"></i>
```

Comment:
Replace `fa` class from font awesome by `fas` since we are using version 5 only

`fa` is now deprecated
https://fontawesome.com/how-to-use/on-the-web/referencing-icons/
![image](https://user-images.githubusercontent.com/11029822/62534565-bcbae880-b849-11e9-9ca0-3a18639434e1.png)